dockerfiles/cpubuilder_ubuntu_jammy_ghr_x86_64.Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt install -y \
         clang-14 lld-14 \
         gcc-9 g++-9 \
         ccache ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
+        ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
```

Comment:
is this supposed to be added?

Reply:
Oh, I meant to just remove "ccache" from that line, since the Dockerfile installs a specific version later using `install_ccache.sh`.

Reply:
sorry, I'm a little confused, ccache isnt in that line

Reply:
Replace
```
        ccache ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
```
with
```dockerfile
        ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
```

Right now you have both lines, so `ninja-build` is listed twice, for example
```
        ccache ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
        ninja-build libssl-dev libxml2-dev libcapstone-dev libtbb-dev \
```

If that gets too confusing or the diffs are too noisy, we can also put these one per line to make such diffs easier to read:
```
        ninja-build \
        libssl-dev \
        libxml2-dev \
        libcapstone-dev \
        libtbb-dev \
```

Reply:
oh i see mb

Reply:
Yep! I'll merge now if that's okay.